addon/lib/metrics.js

```diff
@@ -103,8 +133,10 @@ module.exports = {
   },
 
   updateExperiment: function(addonId, data) {
+    const features = (addonId in store.experimentVariants ?
+                      store.experimentVariants[addonId] : {});
     store.telemetryPingPayload.tests[addonId] = Object.assign(
-      store.telemetryPingPayload.tests[addonId] || {features: {}},
+      store.telemetryPingPayload.tests[addonId] || {features: features},
```

Comment:
Thinking through this & playing with it on my machine, it occurs to me that this won't work quite right.

The `|| {features: features}` part only works on the first initialization, where there's no ping data for an experiment yet. So, if an experiment is already enabled and it later adds or changes variants, this info never gets updated.